src/pages/ClerkScreen.tsx

```diff
@@ -188,6 +251,56 @@ const ClerkScreen = ({
           </Prose>
         )}
       </Sidebar>
+      <Modal
+        isOpen={isDateModalActive}
+        centerContent
+        content={
+          <Prose textCenter>
+            <h1>
+              {twelveHourTime(systemDate.toString())},{' '}
+              {weekdayAndDate(systemDate.toString())}
+            </h1>
+            <div>
+              <InputGroup>
+                <TextInput
+                  type="time"
+                  value={inputTime(systemDate)}
+                  onChange={updateSystemTime}
+                />
+                <TextInput
+                  type="date"
+                  min="2020-10-01"
+                  value={inputDate(systemDate)}
+                  onChange={updateSystemDate}
+                />
+                <Select
+                  id="datetime"
+                  value={systemTimeZone}
+                  onBlur={updateTimeZone}
+                  onChange={updateTimeZone}
+                >
+                  <option value="" disabled>
+                    Select timezone…
+                  </option>
+                  {AMERICA_TIMEZONES.map((timezone) => (
+                    <option key={timezone} value={timezone}>
+                      {timezone.replace(/_/gi, ' ')}
```

Comment:
If we're displaying without underscores here, should we do that on the main screen too?

Reply:
yep, but gonna move to timezone names.